specification/redis/resource-manager/Microsoft.Cache/stable/2021-06-01/redis.json

```diff
@@ -2684,6 +2691,10 @@
           "items": {
             "$ref": "#/definitions/UpgradeNotification"
           },
+          "x-ms-identifiers": [
+            "name",
+            "timestamp"
```

Comment:
Hmm. How do we know timestamp is a good identifier? What if it's not unique?? While it seems unlikely to be duplicated, I don't think we have a particular guarantee of uniqueness.

Reply:
@vukelich Please follow-up. If required, make a new PR for fix.